components/add-layers/vector/add-layers-vector.spec.ts

```diff
@@ -34,6 +35,10 @@ describe('add-layers-vector', () => {
         }
       });
 
+      angular.module('hs.styles', []).service('HsStylerService', function () {
+        this.parseStyle = (new HsStylerService(null)).parseStyle;
```

Comment:
Strange way of doing things 😮

Reply:
I know, but it would no yield otherwise.